multiversion/src/main/scala/multiversion/commands/ExportCommand.scala

```diff
@@ -338,13 +337,7 @@ case class ExportCommand(
   }
   private def withProgressBar[T](renderer: ProgressRenderer)(thunk: => T): T = {
```

Comment:
Would it make sense to move `withProgressBar` to `ProgressBars`?

Reply:
The variable `out` in the new version of the code is unused here, strange that I didn't get a lint warning of some sort 🤔 

The code is now simple enough that I just inlined it into the call site. The update interval duration here is 100ms instead of the default of 16ms. Not sure why; it might be that it takes a little bit longer than 16ms to calculate the progress message on each tick?